spring-cloud-alicloud-ans/src/main/java/org/springframework/cloud/alicloud/ans/endpoint/AnsEndpointAutoConfiguration.java

```diff
@@ -25,7 +26,7 @@
  * @author xiaolongzuo
  */
 @ConditionalOnWebApplication
-@ConditionalOnClass(name = "org.springframework.boot.actuate.endpoint.AbstractEndpoint")
+@ConditionalOnClass(AbstractEndpoint.class)
```

Comment:
why use `AbstractEndpoint.class` here ,but use `@ConditionalOnClass(name = "org.springframework.boot.actuate.endpoint.AbstractEndpoint")` in MigrateEndpointAutoConfiguration

Reply:
goog ,use AbstractEndpoint.class or Endpoint.class